utils/data_utils.py

```diff
@@ -125,7 +139,15 @@ def get_summary(df_exposure, df_adm, adm_level):
     )
     people_exposed_formatted = "{:,}".format(people_exposed)
 
-    return (
-        name,
-        f"{people_exposed_formatted} people exposed to flooding as of {max_date}.",
+    # TODO: Remove need for mapping
+    tercile_label = {-1: "below average", 0: "average", 1: "above average"}
```

Comment:
Tiny thing- we may want to use the word "normal" instead of "average". This is how I've always seen terciles named, and I think it makes more sense since it's not really the average in the mathematical sense.